sklearn/model_selection/_search.py

```diff
@@ -590,8 +590,8 @@ def fit(self, X, y=None, groups=None, **fit_params):
             n_features is the number of features.
 
         y : array-like, shape = [n_samples] or [n_samples, n_output], optional
-            Target relative to X for classification or regression;
-            None for unsupervised learning.
+        Target relative to X for classification or regression; None for
+        unsupervised learning. y can also be a sparse matrix.
```

Comment:
Indentation seems wrong here.

Reply:
Thanks for the comment ! I did not see it... Fixed!